swift/apple/Makefile

```diff
@@ -0,0 +1,11 @@
+PLATFORM_MACOS = macOS,arch=x86_64
+build-macos:
+	@xcodebuild build \
+		-workspace Firezone.xcworkspace \
+		-scheme Firezone \
+		-destination platform="$(PLATFORM_MACOS)"
```

Comment:
```suggestion
		-destination platform="macOS,arch=$(uname -m)"
```

Reply:
Fixed in aa5e12a (Apparently, I need to do `$(shell name -m)`.)